README.md

```diff
@@ -126,8 +128,3 @@ Website: http://rtrlib.realmv6.org/
 
 Mailing List: rtrlib@googlegroups.com
```

Comment:
question: can this be changed to a `mailto` link?!

Reply:
Probably! I will take a quick look into that 👍